lib/protocols/markup.ex

```diff
@@ -0,0 +1,18 @@
+defimpl String.Chars, for: Tuple do
+  def to_string(elem), do: to_markup(elem)
```

Comment:
For what is this implementation required? Don't forget that you are now implementing an entire protocol for all applications that use Tuples, which has a global effect on any application that implements this.